mysite/templates/mysite/project_list.html

```diff
@@ -0,0 +1,30 @@
+{% extends "mysite/base_generic.html" %}
+
+{% block content %}
+
+<h1>Projects</h1>
+{% if object_list %}
+    <ul>
+        {% for project in object_list %}
+            <li>{{ project.name }}  :
+            <a href="{% url "project_view" project.id %}">View</a>
+                {% if user.user_role == 'recruiter' %}
```

Comment:
show there links only when user is recruiter and project is created by that user too.

Reply:
Feedback Incorporated.